langfuse/integrations.py

```diff
@@ -0,0 +1,99 @@
+import os
+import functools
+from datetime import datetime
+from dotenv import load_dotenv
+
+import openai
+from openai.api_resources import ChatCompletion, Completion
```

Comment:
@maxdeichmann we should probably not fully import "lesser known" packages directly to provide more context. Can we change this to `from openai import api_resources` and then call like `api_resources.ChatCompletion`? Just at thought